src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -7765,13 +8255,14 @@ namespace HVACVariableRefrigerantFlow {
         QZnReqTemp = QZnReq;
         if (std::abs(QZnReq) < 100.0) QZnReqTemp = sign(100.0, QZnReq);
         OnOffAirFlowRatio = Par(6);
+        SuppHeatCoilLoad = 0.0;
 
         if (VRF(VRFTU(VRFTUNum).VRFSysNum).VRFAlgorithmTypeNum == AlgorithmTypeFluidTCtrl) {
             // Algorithm Type: VRF model based on physics, appliable for Fluid Temperature Control
-            VRFTU(VRFTUNum).CalcVRF_FluidTCtrl(VRFTUNum, FirstHVACIteration, PartLoadRatio, ActualOutput, OnOffAirFlowRatio);
+            VRFTU(VRFTUNum).CalcVRF_FluidTCtrl(VRFTUNum, FirstHVACIteration, PartLoadRatio, ActualOutput, OnOffAirFlowRatio, SuppHeatCoilLoad);
         } else {
             // Algorithm Type: VRF model based on system curve
-            CalcVRF(VRFTUNum, FirstHVACIteration, PartLoadRatio, ActualOutput, OnOffAirFlowRatio);
+            VRFTU(VRFTUNum).CalcVRF(VRFTUNum, FirstHVACIteration, PartLoadRatio, ActualOutput, OnOffAirFlowRatio, SuppHeatCoilLoad);
```

Comment:
This looks weird. CalcVRF is a function and this makes it look like a member variable. @Myoldmopar is this standard syntax?

Reply:
@rraustad  This is how you call if it is member function. This is not full OO but it is a step in that direction.

Reply:
Right, CalcVRF is a member variable...that just happens to be a function.  So everyone's right.  This is good.